### PR TITLE
refactor(internal): removes logging at trace level

### PR DESCRIPTION
**Pull request by zaidusmani26**

### CODE REVIEW

**Review by Craig-Spencer-12 — APPROVED** [review]
LGTM

File: internal/local/activate.go [hunk 1]
```diff
@@ -136,6 +136,10 @@ func (service *ProvisioningService) GetGeneralSettings() (general.Response, erro
 	if err != nil {
 		return general.Response{}, err
 	}
+	if service.flags.Verbose {
```
Comment (rsdmike) [inline internal/local/activate.go:139]: You shouldn't need the condition -- the log level sets when these should be logged

File: internal/local/activate.go [hunk 2]
```diff
@@ -153,6 +157,10 @@ func (service *ProvisioningService) HostBasedSetup(digestRealm string, password
 	if hostBasedSetupResponse.Body.Setup_OUTPUT.ReturnValue != 0 {
 		return utils.ActivationFailed, errors.New("unable to activate CCM, check to make sure the device is not alreacy activated")
 	}
+	if service.flags.Verbose {
```
Comment (rsdmike) [inline internal/local/activate.go:160]: Remove condition

File: internal/local/activate.go [hunk 3]
```diff
@@ -167,6 +175,10 @@ func (service *ProvisioningService) GetHostBasedSetupService() (hostbasedsetup.R
 	if err != nil {
 		return hostbasedsetup.Response{}, err
 	}
+	if service.flags.Verbose {
```
Comment (rsdmike) [inline internal/local/activate.go:178]: Remove condition

File: internal/local/activate.go [hunk 4]
```diff
@@ -390,6 +402,10 @@ func (service *ProvisioningService) sendAdminSetup(digestRealm string, nonce []b
 			return utils.UnableToActivate, err
 		}
 	}
+	if service.flags.Verbose {
```
Comment (rsdmike) [inline internal/local/activate.go:405]: Remove condition

File: internal/local/activate_test.go [hunk 5]
```diff
@@ -68,7 +68,7 @@ func TestActivation(t *testing.T) {
 
 func TestActivateCCM(t *testing.T) {
 	f := &flags.Flags{}
-
+	f.Verbose = true
```
Comment (rsdmike) [inline internal/local/activate_test.go:71]: I don't think we turn this on in testing as will put a lot of noise into the console.

**Review by rsdmike — CHANGES_REQUESTED** [review]

**Review by rsdmike — APPROVED** [review]

**Pull request merged**